apps/system/utils/netcmd_netmon.c

```diff
@@ -44,75 +44,63 @@
 /****************************************************************************
  * Preprocessor Definitions
  ****************************************************************************/
-#define USAGE							\
-	"\n usage: netmon [options]\n"		\
-	"\n socket information:\n"			\
-	"       netmon sock\n"				\
-	"\n WiFi Manager stats:\n"			\
-	"       netmon wifi\n"				\
-	"\n Net device stats:\n"			\
+#define USAGE									\
+	"\n usage: netmon [options]\n"				\
+	"\n socket information:\n"					\
+	"       netmon sock\n"						\
+	"\n WiFi Manager stats:\n"					\
+	"       netmon wifi\n"						\
+	"\n Net device stats:\n"					\
 	"       netmon [devname]\n\n"
 
 #ifdef CONFIG_NET_IPv6
-#define PRINT_IPV(sock)										\
-	do {													\
-		if (sock->type & NETMON_TYPE_IPV6) {				\
-			printf("IPv6\t");								\
-			printf("%-15s:", _netmon_ip6addr_ntoa((const ip6_addr_t*)&sock->local_ip));	\
-			if (sock->local_port == 0) {					\
-				printf("%-8s", "*");						\
-			} else {										\
-				printf("%-8d", sock->local_port);			\
-			}												\
-			printf("%-15s:", _netmon_ip6addr_ntoa((const ip6_addr_t*)&sock->remote_ip));	\
-			if (sock->remote_port == 0) {					\
-				printf("%-8s", "*");						\
-			} else {										\
-				printf("%-8d", sock->remote_port);			\
-			}												\
-		} else {											\
-			printf("IPv4\t");								\
-			printf("%-15s:", inet_ntoa(sock->local_ip));	\
-			if (sock->local_port == 0) {					\
-				printf("%-8s", "*");						\
-			} else {										\
-				printf("%-8d", sock->local_port);			\
-			}												\
-			printf("%-15s:", inet_ntoa(sock->remote_ip));	\
-			if (sock->remote_port == 0) {					\
-				printf("%-8s", "*");						\
-			} else {										\
-				printf("%-8d", sock->remote_port);			\
-			}												\
-		}													\
-	} while (0)
+static inline void PRINT_IPV(struct netmon_sock *sock)
+{
+	if (sock->type & NETMON_TYPE_IPV6) {
+		printf("IPv6 (not yet supported)\t");
```

Comment:
Why did you remove to print information about IPv6?

Reply:
monitoring IPv6 can't be provided with current bsd socket. it can be added after bsd socket API supports ipv6